pkg/apis/kubeone/types.go

```diff
@@ -51,6 +51,8 @@ type KubeOneCluster struct {
 	DynamicWorkers []DynamicWorkerConfig `json:"dynamicWorkers,omitempty"`
 	// MachineController configures the Kubermatic machine-controller component.
 	MachineController *MachineControllerConfig `json:"machineController,omitempty"`
+	// CABundle PEM encoded global CA
+	CABundle string `json:"caBundle,omitempty"`
```

Comment:
I'd say that this should be a path to the bundle on local FS instead of the bundle itself. The bundle can be huge (the Mozilla one is 3k+ lines).

Reply:
While yes, the original public CA bundle is long, this field should only be used in case when you have you very own custom CA bundle, which most likely will not be that long.

Reply:
Then let's build on top of this PR as it's already quite big. My proposal would be to retrofit the "special content" linking. 
I.e. let's have `file:///path/to/file` in `.CABundle` and in case if it starts from `file://` that will be interpreted as reference to the file, instead of actual content.

Reply:
This sounds like a good plan. Can you create a follow-up issue for this?